examples/maps/maps.description

```diff
@@ -1,5 +1,5 @@
-// The `map<T>` type is a `map` from strings to `T`. `map` syntax is similar to JSON. Maps are mutable: `m[k]` is an
-// `lvalue`. `foreach` will iterate over the values of the `map`. `m[k]` gets entry for `k`; `nil` if missing.
-// Use `m.get(k)` when you know that there is an entry for `k`. `m.keys()` can be used to iterate over keys,
-// to get the keys as an `array` of strings. `==` and `!=` on maps is deep: two maps are equal if they
-// have the same set of keys and the values for each key are equal.
+// The `map<T>` type is a `map` from strings to `T`. The `map` syntax is similar to JSON. Maps are mutable.
```

Comment:
is a mapping? 🤔 Wondering cz when "map" is formatted as code, it naturally makes one think of the `map` type descriptor.